chaoslib/chaoskube/node_freeze_chaos.yml

```diff
@@ -0,0 +1,80 @@
+---
+- block:
+
+   - name: Setup node freeze infra
+     shell: >
+       kubectl create -f {{ daemonset }} -n {{ namespace }}
+     args:
+       executable: /bin/bash
+     register: result
+    
+   - name: Confirm that node freeze chaos ds is running on all nodes
+     shell: >
+       kubectl get pod -l app=node-freeze
+       --no-headers -o custom-columns=:status.phase
+       -n {{ namespace }} | sort | uniq
+     args: 
+       executable: /bin/bash
+     register: result
+     until: "result.stdout == 'Running'"
+     delay: 20
+     retries: 15
+
+   - name: Obtaining the application pod name using its label.
+     shell: >
+       kubectl get pods -n {{ namespace }} -l {{ app_label }} | grep
+       -w Running | awk '{print $1}' | head -1 
+     args:
+       executable: /bin/bash
+     register: app_pod
```

Comment:
It's good to get app pod name once again in other utils and It will be better If we reduce redundancy using `set_fac` in utils. It's your wish to change it or not.

Reply:
@mittachaitu this test is actually comprises of data-persistency test which is first obtaining the application pod name , if anyone   run the node-freeze test alone  you will require application name fetch task in its utils also.thats why i left that tasks untouched.